bin/pycbc_banksim

```diff
@@ -374,7 +374,7 @@ with ctx:
                                       template_sample_rate,
                                       filter_N, options.filter_sample_rate)
 
-            h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
+                h_norm = sigmasq(htilde, psd=psd, low_frequency_cutoff=f_lower,
```

Comment:
If I remember well it was added/used by @alex-nielsen for some studies related to NSBH signals with tidal disruption, where the early cutoff of the signal was approximated with a simple truncation. Let me check with him just in case.